Tweak.xm

```diff
@@ -332,6 +350,42 @@ YTHotConfig *(*InjectYTHotConfig)();
 
 // %end
 
+#import <MediaRemote/MediaRemote.h>
+
+#pragma mark - this functions are responsible for breaking some stuff when dismissing
+#pragma mark - PIP or using MRMediaRemoteCommandPause, so we need to find a way to pause
+#pragma mark - the playback without calling those functions in the process.
+// %hook MLPIPController
+// - (void)pause {
+// }
+// %end
+// %hook AVSampleBufferDisplayLayerPlayerController
+// - (void)setPaused:(BOOL)arg1 {
+// }
+// %end
```

Comment:
Remove unused code.

Reply:
I did not add this part of the code; I thought this is part of making it iOS 13 copatible?